vault/core.go

```diff
@@ -2320,12 +2322,15 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 	if err := c.setupHeaderHMACKey(ctx, false); err != nil {
 		return err
 	}
-	if err := c.runLockedUserEntryUpdates(ctx); err != nil {
-		return err
-	}
-	c.updateLockedUserEntries()
-
 	if !c.IsDRSecondary() {
+		if err := c.runLockedUserEntryUpdates(ctx); err != nil {
```

Comment:
Why do we runLockedUserEntryUpdates here, why not rely solely on the invocations in updateLockedUserEntries?

Reply:
runLockedUserEntryUpdates does two things:
1. It updates the storage entries i.e; if there are any stale entries it removes them
2. It updates the userFailedLoginInfo map with locked user values upon unseal

Since updateLockedUserEntries runs every 15 minutes, this function runs once upon unseal to update the  userFailedLoginInfo map.

Reply:
Got it, ty.  I was thinking it could be moved into updateLockedUserEntries, but it's not a big deal.

Reply:
I'll resolve this since Hamid already has a comment to that effect.

Reply:
Moved runLockedUserEntryUpdates inside updateLockedUserEntries with latest commit